addons/mail/static/src/core/common/message_reaction_list.scss

```diff
@@ -0,0 +1,19 @@
+.o-mail-MessageReactionList {
+    max-width: 250px;
+    overflow: hidden;
+    word-break: break-word;
+}
+
+.o-mail-MessageReaction-Summary {
+    cursor: pointer;
+
+    &.active {
```

Comment:
When you can, prefer style with classname, e.g. `.cursor-pointer`, `.overflow-hidden`